.github/workflows/test.yml

```diff
@@ -42,7 +42,14 @@ jobs:
           - linkerd
           - nginx-ingress
           - openshift
-          - other
+          # - other
+          - other/a
+          - other/b-d
+          - other/e-l
+          - other/m-p
+          - other/rec-req
+          - other/res
+          - other/s-v
```

Comment:
Please rename this folder `s-z` both here in the workflow and the actual folder. Even though there aren't currently policies which begin with letters after `v`, this is where they would go.